### PR TITLE
Setup work to support style utility components

### PR DESCRIPTION
**Pull request by mperrotti**
There are no visual changes, these are just changes to lower level code (like design tokens) to support the other style utility PRs.

Just check that tests still pass and the code looks good. See other style utility PRs for additional context

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
`npm run dist` is failing locally, as well as in CI.
Screenshot from 2019-07-04 10-00-49

File: package-lock.json [hunk 1]
```diff
@@ -10437,9 +10437,9 @@
       }
     },
     "csstype": {
-      "version": "2.6.3",
```
Comment (GeorgiSTodorov) [inline package-lock.json]: Why did we have csstype in package-lock if we are only adding it now?

Comment (pierrebeitz) [reply package-lock.json]: it's a dependency of `@emotion`!

Comment (GeorgiSTodorov) [reply package-lock.json]: That makes sense, I guess.

Comment (pierrebeitz) [reply package-lock.json]: here's a boiled down section of the `package-lock.json` that hopefully makes some sense:

```
```

**Pull request merged**